piet-direct2d/src/lib.rs

```diff
@@ -425,6 +425,10 @@ impl<'a> RenderContext for D2DRenderContext<'a> {
             .set_transform(&affine_to_matrix3x2f(self.current_transform()));
     }
 
+    fn current_transform(&self) -> Affine {
+        self.current_transform()
```

Comment:
can we just move the other function down here, or does it need to be used in contexts where we don't have access to the trait?